README.md

```diff
@@ -14,10 +14,14 @@ See the [OpenTelemetry Instrumentation for Python](https://opentelemetry.io/docs
 | ------- | ------------ | ------- |
 | Traces  | Stable       | N/A     |
 | Metrics | Stable       | N/A     |
-| Logs    | Experimental | N/A     |
+| Logs    | Experimental* | N/A     |
 
 Project versioning information and stability guarantees can be found [here](./rationale.md#versioning-and-releasing).
 
+***Breaking Changes**
```

Comment:
Same comment https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts